src/Tempest/Http/src/Mappers/PsrRequestToRequestMapper.php

```diff
@@ -24,6 +25,8 @@ public function canMap(mixed $from, mixed $to): bool
     public function map(mixed $from, mixed $to): array|object
     {
         /** @var PsrRequest $from */
+        $from = (new JsonParserServerRequest())($from);
```

Comment:
I find the naming a bit odd here. This is an action class that makes a `ServerRequestInterface`, couldn't it be called `ParseJsonServeRequest` or something alike?

Also, is there a reason you opted for making this a class instead of a private method?